src/main/java/frc/robot/constants/RobotConfig.java

```diff
@@ -4,4 +4,34 @@
  * Software config settings (e.g. max speed, PID values). For hardware constants @see
  * RobotConstants"
  */
-public class RobotConfig {}
+public class RobotConfig {
+  public static final class IntakeConfig {
```

Comment:
Why do we need this?

Reply:
Intake config formerly held PID constants. After PID constants were removed, all that's stored in there is Default Speed and Timeout time